requirements/customer/scheduling-algorithm.tex

```diff
@@ -0,0 +1,48 @@
+\subsection{Scheduling Algorithm}
+This section describes how the central server distributes jobs to work machines: the \textit{scheduling algorithm}.
+Put briefly, the scheduling algorithm takes a list of jobs and decides which ones to run on the available work machines.
+The scheduling algorithm has the following goals:
+\begin{itemize}
+\item Keep work machines busy: there should be no idle work machines if there are pending jobs.
```

Comment:
This sounds a little bit misleading to me. There can be pending jobs with requirements unmatched by the idle machines.